crates/bevy_scene/src/dynamic_scene.rs

```diff
@@ -10,10 +10,11 @@ use bevy_reflect::{Reflect, TypeRegistryArc, TypeUuid};
 
 #[cfg(feature = "serialize")]
 use crate::serde::SceneSerializer;
+use bevy_ecs::reflect::ReflectResource;
 #[cfg(feature = "serialize")]
 use serde::Serialize;
 
-/// A collection of serializable dynamic entities, each with its own run-time defined set of components.
+/// A collection of serializable resources and dynamic entities, each of the latter with its own run-time defined set of components.
```

Comment:
Nit: I think the second phrase of "each of the latter" is a little confusing. Maybe split that part off into a separate sentence?
```suggestion
/// A collection of serializable resources and dynamic entities.
///
/// Each dynamic entity in the collection contains its own run-time defined set of components.
```